anomaly_detector/anomaly_detector.py

```diff
@@ -188,7 +188,7 @@ def infer(self, false_positives=None):
                 s["anomaly_score"] = dist[i]
                 # Record anomaly event in fact_store and
 
-                if dist[i] > threshold and {"message": s["message"]} not in false_positives:
```

Comment:
This was removed because it caused the system to crash due to no messages found in false positive or it being Null. This will be fixed in #136 issue and in a separate PR.